become-a-committer.md

```diff
@@ -0,0 +1,36 @@
+# Become a Committer
```

Comment:
This should be outlined in our https://github.com/tikv/tikv/blob/master/GOVERNANCE.md, a file which must exist due to the requirements of the CNCF, or https://tikv.org/docs/3.0/contribute/contribute-to-tikv/.